templates/azure/main.tf.j2

```diff
@@ -1,5 +1,11 @@
 terraform {
   required_version = ">= 0.12"
+  required_providers {
+    az = {
```

Comment:
Please change it to `azurerm`:

https://www.terraform.io/docs/language/providers/requirements.html#local-names

> Users of a provider can choose any local name for it. However, nearly every provider has a preferred local name, which it uses as a prefix for all of its resource types. (For example, resources from hashicorp/aws all begin with aws, like aws_instance or aws_security_group.)
> 
> Whenever possible, you should use a provider's preferred local name. This makes your configurations easier to understand, and lets you omit the provider meta-argument from most of your resources. (If a resource doesn't specify which provider configuration to use, Terraform interprets the first word of the resource type as a local provider name.)